README.md

```diff
@@ -36,7 +36,7 @@ If you discover an issue related to the Android Gradle Plugin, please file an is
 
 This plugin should be applied anywhere the `com.android.application` or `com.android.library` plugins are applied. We recommend adding the plugin to your project's [conventions plugin](https://docs.gradle.org/current/samples/sample_convention_plugins.html).
 
-<details open>
+<details>
 <summary>Groovy</summary>
```

Comment:
I would also switch the order and put the Kotlin sample first. Please do this for all the samples.

Reply:
I was in the process of typing the same thing.

I'm so excited to see this is finally happening!

Reply:
done [8a4e48a](https://github.com/gradle/android-cache-fix-gradle-plugin/pull/488/commits/8a4e48a9b1d6c81a36db8c7973ec38696e6a3498)